docfx_project/ziti/clients/linux.md

```diff
@@ -1,59 +1,147 @@
+
+```bash
+# shell script illustrating the steps to install or upgrade ziti-edge-tunnel
+wget -q "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/ziti-edge-tunnel-Linux_$(uname -p).zip" \
+  && unzip ./ziti-edge-tunnel-Linux_$(uname -p).zip \
+  && rm ./ziti-edge-tunnel-Linux_$(uname -p).zip \
+  && chmod -c +x ./ziti-edge-tunnel \
+  && ./ziti-edge-tunnel version
+```
+
+Optionally, you may install `ziti-edge-tunnel.service` as a systemd unit. For example,
+
+```ini
+# /usr/lib/systemd/system/ziti-edge-tunnel.service
+[Unit]
+Description=Ziti Edge Tunnel
+After=network-online.target
+
+[Service]
+Type=simple
+EnvironmentFile=/opt/openziti/etc/ziti-edge-tunnel.env
+ExecStart=/opt/openziti/bin/ziti-edge-tunnel run --verbose=${ZITI_VERBOSE} --dns-ip-range=${ZITI_DNS_IP_RANGE} --identity-dir=${ZITI_IDENTITY_DIR}
+Restart=always
+RestartSec=3
+
+[Install]
+WantedBy=multi-user.target
+```
+
+
+
+### Enroll Before You Run
+
+You will need the token file or its contents to enroll. Enrollment is the act of exchanging the token for an identity that is subsequently installed permanently in the filesystem.
+
+```bash
+# enroll from a token file
+./ziti-edge-tunnel enroll --jwt ./myTunneler.jwt --identity ./myTunneler.json
+```
+
+```bash
+# enroll from stdin
+./ziti-edge-tunnel enroll --jwt - --identity ./myTunneler.json < ./myTunneler.jwt
+```
+
+### Global Options
 
-`ziti-tunnel` fails to start if no intercept modes can be successfully initialized:
 
-    $ sudo ziti-tunnel run ziti.json
-    [   0.000]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: tproxy initialization failed: failed to initialize iptables handle: exec: "iptables": executable file not found in $PATH
-    [   0.001]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: tun initialization failed: failed to open tun interface (name='', mtu=0): ioctl failed with 'invalid argument'
-    [   0.001]   FATAL ziti/tunnel/cmd/ziti-tunnel/subcmd.run: failed to initialize an interceptor
+```bash
+# Load a single identity.
+--identity <identity>
+```
+
+```bash
+# Load all identities in a dir, ignoring files with a .bak or .original filename suffix.
+--identity-dir <dir>
+```
 
-### Intercept Modes
+```bash
+# Set log level, higher is more verbose (default level 3 means INFO).
+--verbose N
+```
+
+```bash
+# Set service polling interval in seconds (default 10).
+--refresh N
+```
+
+### Run Mode
+
+`ziti-edge-tunnel run` provides a transparent proxy and nameserver. The nameserver may be configured to be authoritative (the default) or recursive with a command-line option. The OS is automatically configured to treat the nameserver as primary. You may inspect the resulting configuration with these commands.
+
+```bash
+resolvectl dns     # inspect the association of tun device and nameserver

```

Comment:
this is the first time resolvectl is referenced right? shouldn't it be somewhere in here that it's required?

Reply:
It's not required any longer by `ziti-edge-tunnel`, but useful for inspecting the configuration. There is yet a fallback behavior in ZET to shell out to that CLI, but only if it was built with a non-default option to disable the default behavior.